client/jetpack-cloud/sections/partner-portal/issue-multiple-licenses-form/index.tsx

```diff
@@ -154,6 +155,7 @@ export default function IssueMultipleLicensesForm( {
 
 			{ ! isLoadingProducts && (
 				<>
+					<QueryProductsList type="jetpack" currency="USD" />
```

Comment:
@yashwin, I just wanted to ask why we override the currency here. Is it because we are offering only our products in USD?

Reply:
This is how we do it on the prices page. So, I wanted to keep it consistent.